spec/requests/physical_servers_spec.rb

```diff
@@ -40,6 +49,33 @@
 
         expect(response).to have_http_status(:success)
       end
+
+      it "immediately restarts a server successfully" do
+        ps = FactoryGirl.create(:physical_server)
+
+        api_basic_authorize action_identifier(:physical_servers, :restart_now, :resource_actions, :post)
+        run_post(physical_servers_url(ps.id), gen_request(:restart_now))
+
+        expect(response).to have_http_status(:success)
```

Comment:
while I see these expectations here match the rest of the file, I think they should all check for the action result signature, looking at others, maybe they should all be something like:

```
          expected = { 
            "message"   => "Requested server state restart_now for Server instance ...",
            "success"   => true,
            "task_href" => a_string_matching(tasks_url),
            "task_id"   => anything
          }   
          expect(response.parsed_body).to include(expected)
          expect(response).to have_http_status(:ok)
```

Reply:
Other than forbidden, because these are action results, failures still come back as 200, but the "success" => false in the resultant hash. This is done for allowing clients deal with bulk requests. Thus the recommendation above.

Reply:
@abellotti I updated the test cases to also check the response body for success to address situations such as the one you described.